lib/bel/translator/plugins/rdf/translator.rb

```diff
@@ -94,6 +94,16 @@ def write_rdf_to_io(format, objects, io, options = {})
         :stream => true
       )
 
+      if (@format == :turtle)
+        require 'yaml'
```

Comment:
Move the require to the top of the file. If we remove the condition above then we'll always need to load the yaml module.